doc/dnf5_plugins/repoclosure.8.rst

```diff
@@ -57,22 +57,17 @@ Options
     Check closure for this package only.
 
 
---------
 Examples
---------
-
-Display list of unresolved dependencies for all enabled repositories::
-
-    dnf5 repoclosure
-
-Display list of unresolved dependencies for rawhide repository and packages with architecture noarch and x86_64::
-
-    dnf5 repoclosure --repo rawhide --arch noarch --arch x86_64
+========
 
-Display list of unresolved dependencies for zmap package from rawhide repository::
+``dnf5 repoclosure``
+    Display list of unresolved dependencies for all enabled repositories.
 
-    dnf5 repoclosure --repo rawhide zmap
+``dnf5 repoclosure --repo rawhide --arch noarch --arch x86_64``
+    Display list of unresolved dependencies for rawhide repository and packages with architecture noarch and x86_64.
```

Comment:
I believe this is more correct.

Display *a* list of unresolved dependencies...

See below too

Reply:
Thanks, fixed.